amy/trainings/forms.py

```diff
@@ -78,6 +79,16 @@ class Meta:
     class Media:
         js = ("trainingprogress_form.js",)
 
+    def add_error(self, field: str, error: ValidationError | str):
+        """Overrides add_error to ignore any errors that are intended to appear
+        on the trainee-only "trainee_notes" field."""
+        if field == "trainee_notes":
+            return
+        elif hasattr(error, "error_dict") and "trainee_notes" in error.error_dict:
+            error.error_dict.pop("trainee_notes")
+
+        super().add_error(field, error)
```

Comment:
This override is a bit of a hack, as I couldn't figure out how to ignore/hide validation errors on `trainee_notes` otherwise while still displaying the field. Alternative suggestions welcome.
(I prefer to hide the error so we aren't showing a "This field is required" error on a disabled field)

Reply:
@elichad Where does this error originate from? Is it the `clean` method in `TrainingProgress` model?

Reply:
If it does, we could move it out and only implement it in the corresponding clean method for the form that is using this validation.

Reply:
Oh, good point. I'll do that.